app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
I know we don't use this here, but... if we manually remove it, I understand we can only generate views in devise based on module scope (eg all views in registrations) which might then override our styled new view. Probably a question for Thomas.